unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/Tabs/FriendsTabView.cs

```diff
@@ -48,12 +68,18 @@ public override bool UpdateEntry(string userId, FriendEntryBase.Model model)
         {
             offlineFriendsList.Remove(userId);
             onlineFriendsList.Add(userId, entry);
+
+            var removedTimestamp = offlineFriendsList.RemoveLastTimestamp(userId);
+            onlineFriendsList.AddOrUpdateLastTimestamp(removedTimestamp);
         }
 
         if (model.status == FriendsController.PresenceStatus.OFFLINE)
         {
             onlineFriendsList.Remove(userId);
             offlineFriendsList.Add(userId, entry);
+
+            var removedTimestamp = onlineFriendsList.RemoveLastTimestamp(userId);
+            offlineFriendsList.AddOrUpdateLastTimestamp(removedTimestamp);
```

Comment:
in these 2 `AddOrUpdateLastTimestamp()` we should re-order the entries too? (as we are doing now) or it's not necessary?

Reply:
I think in in this case it is always necessary to reorder because the user have just changed his online status (he changes his place from a tab to other in the view) and we don't know which users will be in the destination list.